### ISSUE TITLE
A query regarding Image-Cap

### ISSUE THREAD

**Issue by rupeshS9102** [ne7ermore/torch_light#10]
https://github.com/ne7ermore/torch-light/tree/master/Image-Cap
Hi @ne7ermore,
I found this code very educational. Thank you. But, I do not have a GPU. When I run train.py on a CPU, the estimated time of training is about 3-4 weeks. So, could you please share the trained model. It would be of great help. My email id - rupesh.s9102@gmail.com.
Thank You

**Comment by Michael-Hsu** [comment]
> https://github.com/ne7ermore/torch-light/tree/master/Image-Cap
> Hi @ne7ermore,
> I found this code very educational. Thank you. But, I do not have a GPU. When I run train.py on a CPU, the estimated time of training is about 3-4 weeks. So, could you please share the trained model. It would be of great help. My email id - [rupesh.s9102@gmail.com](mailto:rupesh.s9102@gmail.com).
> Thank You

Hello, did you get the trained model? I also have the same problem. If you do get it, could you please share the trained model with me ? My email is broadenhsu@gmail.com. Thanks a lot.

**Comment by Michael-Hsu** [comment]
@ne7ermore 
Thanks for your sharing code for Image Cap. I find there might be some bug in the train.py. There isn't definition of "speak" in model.py. Or I misunderstand, could you please give me some advice?

def eval():
    actor.eval()
    eval_score = .0
    for imgs, labels in tqdm(validation_data,
                             mininterval=1,
                             desc="Actor-Critic Eval",
                             leave=False):
        enc = actor.encode(imgs)

        hidden = actor.feed_enc(enc)
        ### Bug to fix??
        words, _ = actor.speak(hidden)

        scores = rouge_l(words, labels)
        scores = scores.sum()

        eval_score += scores.data

    eval_score = eval_score[0] / validation_data.sents_size

    return eval_score

**Comment by fireholder** [comment]
I met the same issue as you, have you solved it ? @Michael-Hsu

**Comment by Michael-Hsu** [comment]
no, do u have any idea? Michael Hsu 邮箱：broadenhsu@gmail.com 签名由 网易邮箱大师 定制 On 10/28/2019 16:00, xwt wrote: I met the same issue as you, have you solved it ? @Michael-Hsu — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.
